### PR TITLE
feat: add orgUnitField to Map View

### PR DESCRIPTION
**Pull request by vietnguyen**
https://jira.dhis2.org/browse/DHIS2-12734

### CODE REVIEW

File: dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_39__Add_organisationUnitCoordinateField_column_to_mapView_table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+alter table mapview add column organisationunitcoordinatefield varchar(255);
```
Comment (jbee) [inline dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_39__Add_organisationUnitCoordinateField_column_to_mapView_table.sql:1]: `if not exists` is missing

**Review by jbee — COMMENTED** [review]
Just the SQL needs the `if not exists` - otherwise good to go.

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/mapping/MapView.java [hunk 2]
```diff
@@ -187,6 +187,8 @@
 
     private String eventCoordinateField;
 
+    private String organisationUnitCoordinateField;
```
Comment (larshelge) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/mapping/MapView.java:190]: Actually we have a field already @vietnguyen , if you look at `BaseAnalyticalObject` we have a field `orgUnitField` already. So you can just map that `orgUnitField` field in the hibernate mapping but not add this field to `MapView`.

Comment (vietnguyen) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/mapping/MapView.java]: fixed

File: dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_39__Add_organisationUnitCoordinateField_column_to_mapView_table.sql (hunk 1 above)
Comment (vietnguyen) [reply dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_39__Add_organisationUnitCoordinateField_column_to_mapView_table.sql]: fixed

**Review by jbee — APPROVED** [review]

**Pull request merged**